s30566-Daniel_Makowski/4. Kolokwium - powtórzenie/Zadanie 4.5.txt

```diff
@@ -0,0 +1,3 @@
+Jak wypisać wszystkie pliki w bieżącym katalogu, których nazwa zaczyna się od al?
+
+	Moja propozycja to: find ./ -name "al*"
```

Comment:
Prawie dobrze. To zwróci wszystko w danym drzewie katalogów wraz z podkatalogami pasujące do wzorca a w tym także i linki symboliczne, katalogi, urządzenia, itd...
Trzeba to trochę ograniczyć ale idea jest ok.